src/OrbitGl/App.cpp

```diff
@@ -2042,6 +2043,19 @@ void OrbitApp::LoadPreset(const PresetFile& preset_file) {
   });
 }
 
+void OrbitApp::ShowPresetInExplorer(const PresetFile& preset) {
+  const auto file_exists = orbit_base::FileExists(preset.file_path());
+  if (file_exists.has_error()) {
+    ERROR("Unable to find preset file: %s", file_exists.error().message());
+    return;
+  }
+
+  std::string command = absl::StrFormat("explorer.exe /select,\"%s\"", preset.file_path().string());
+  if (system(command.c_str()) == 0) return;
```

Comment:
Could you add a comment to explain this line?

Reply:
Using `system` works but is not great from a security perspective as it passes the command string to cmd.exe (to /bin/sh on Linux). There might be ways to inject commands into the preset name which we want to avoid at all costs.

It would be better to use `QProcess::execute` instead. As far as I remember we have a Qt dependency in OrbitGl now anyway, right?

Reply:
Thanks for the suggestion! Changed to use `QProcess::startDetached` here and added a comment.